src/util/Constants.ts

```diff
@@ -261,6 +261,20 @@ export const CommandResponses = {
 			.setDescription(desc)
 			.setFooter('Sentinel | <> = Not Required | [] = Required')
 			.setTimestamp();
+	},
+	SUGGESTION_RESPONSE: () => 'Thank you for your suggestion, my developers will look into this!',
+	SUGGESTION_LOG: (user: User, content: string) => {
+		return new MessageEmbed()
+			.setColor(SentinelColors.LIGHT_BLUE)
+			.setAuthor('Sentinel New Suggestion')
+			.setTimestamp()
+			.addFields({
+				name: 'Message Author',
+				value: user.tag
```

Comment:
```suggestion
				value: formatUser(user)
```
the user needs to be formatted, so it shows the ID and such